packaging/manpage.md

```diff
@@ -0,0 +1,39 @@
+% ccx_preCICE(1) 2.17
+% Boris Martin
+% Oct 2021
```

Comment:
How is this information processed?
The official guideline to writing man pages state that one should avoid the author block.

Reply:
It was part of an unofficial template I used, I didn't really pay attention to.
Output on my machine is this : 

![image](https://user-images.githubusercontent.com/84379125/141147905-9d8750ef-9372-4cd2-9e3f-778ceccadbee.png)

No objection te removing it if you think it's better.

Reply:
The official man page from man-pages on the convention for writing linux man pages :exploding_head: highly suggests to omit the AUTHOR section. https://linux.die.net/man/7/man-pages